### ISSUE TITLE
App initially renders too tall in Android chromium-based mobile browsers.

### ISSUE THREAD

**Issue by joelc0193** [flutter/flutter#145644]
### Steps to reproduce

1) Start a new flutter project
2) Create a widget that takes up the entire height of the screen.
3) Run a local server
4) Open a new tab in an Android chromium-based browser on a mobile device and navigate to the locally running flutter webapp.

### Expected results

The widget should be exactly as tall as the screen.

### Actual results

The widget is taller than the screen. Only once you tap it does it adjust its height to match the height of the screen.

### Code sample

<details open><summary>Code sample</summary>

```dart
class HomePage extends StatefulWidget {
 @override
 State<HomePage> createState() => _HomePageState();
}


class _HomePageState extends State<HomePage> {
 @override
 Widget build(BuildContext context) {
   return Container(
     decoration: BoxDecoration(
       color: Colors.black,
       border: Border.all(
         color: Colors.red,
         width: 3.0,
       ),
     ),
   );
 }
}



```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots </summary>

![BeforeTap](https://github.com/flutter/flutter/assets/20515566/21de6a28-e888-41b2-a28f-20cb57abcd07)
![AfterTap](https://github.com/flutter/flutter/assets/20515566/d6112e04-8248-493a-86e9-0c443e7646c4)

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
Joels-MBP:poster_fans jcastro2$ flutter doctor -v
[✓] Flutter (Channel stable, 3.10.6, on macOS 14.2.1 23C71 darwin-x64, locale en-US)
    • Flutter version 3.10.6 on channel stable at /Users/jcastro2/fvm/versions/3.10.6
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f468f3366c (9 months ago), 2023-07-12 15:19:05 -0700
    • Engine revision cdbeda788a
    • Dart version 3.0.6
    • DevTools version 2.23.1

[✓] Android toolchain - develop for Android devices (Android SDK version 35.0.0-rc1)
    • Android SDK at /Users/jcastro2/Library/Android/sdk
    • Platform android-34, build-tools 35.0.0-rc1
    • ANDROID_HOME = /Users/jcastro2/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.7+0-17.0.7b1000.6-10550314)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 15E204a
    • CocoaPods version 1.15.2

[✓] Chrome - develop for the web
    • CHROME_EXECUTABLE = /Applications/Brave Browser.app/Contents/MacOS/Brave Browser

[!] Android Studio (version 2023.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[!] Android Studio (version 2023.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[✓] VS Code (version 1.87.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.84.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 14.2.1 23C71 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Brave Browser 122.1.63.174

[✓] Network resources
    • All expected network resources are available.

! Doctor found issues in 2 categories.
Joels-MBP:poster_fans jcastro2$ 
```

</details>

**Comment by danagbemava-nc** [comment]
Hi @joelc0193, you are using an older version of flutter. Kindly upgrade to the latest version flutter stable (3.19.4) to see if you still experience this issue. If you do, please provide the updated output of flutter doctor -v. 

Thank you

**Comment by joelc0193** [comment]
Same issue still even with Flutter 3.19.4

```
^[[AJoels-MBP:test jcastro2$ flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.19.4, on macOS 14.2.1 23C71 darwin-x64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 35.0.0-rc1)
[✓] Xcode - develop for iOS and macOS (Xcode 15.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2023.1)
[✓] VS Code (version 1.87.2)
[✓] Connected device (2 available)
[✓] Network resources

• No issues found!

Joels-MBP:test jcastro2$ 
```

@danagbemava-nc

**Comment by danagbemava-nc** [comment]
Closing this as a duplicate of https://github.com/flutter/flutter/issues/145667. 

Kindly subscribe to the issue linked above for updates. 

Thank you
